pkg/cluster/executor/executor.go

```diff
@@ -23,17 +23,20 @@ import (
 	"github.com/pingcap/tiup/pkg/localdata"
 )
 
+// SSHType represent the type of the chanel used by ssh
+type SSHType = string
```

Comment:
```suggestion
type SSHType string
```
I think it should be a new type instead of a type alias.

Reply:
If use `type SSHType string`, the SSHType filed in gOpt should be
```
SSHType           string
```
instead of
```
SSHType           executor.SSHType
```
Because cobra desn't accept SSHType as params
And everywhere we pass it to executor.New must use force cast, is that OK?

Reply:
`.StringVar((*string)(&gOpt.SSHType))`